config/locales/en.yml

```diff
@@ -645,16 +645,21 @@ en:
       email_body: |
         Hello %{preferred_name},
 
-        Thanks for starting your taxes with GetYourRefund - you’re almost there! We will prepare your taxes once we have all of your required information and tax documents.
+        Thanks for starting your taxes with GetYourRefund - you’re almost there! Your case number is %{client_id}. We will prepare your taxes once we have all of your required information and tax documents.
 
-        If you have more documents to send or have any questions about your case you can email your tax preparer at hello@getyourrefund.org, chat with our specialists using the chat box on getyourrefund.org/tax-questions or submit more documents at your secure submission link: %{requested_docs_link}
+        You can securely upload your ID and additional tax documents here: %{requested_docs_link}
+
+        If you have any questions you can contact your tax team via email <a href="mailto:hello@getyourrefund.org>hello@getyourrefund.org</a> or text message 58750.
 
         Remember, we’ll review everything with you before filing so just do the best you can. Then we’ll help you get the tax credits that belong to you!
 
         We’re here to help!
-        Your tax team at GetYourRefund.org
+        Your Tax Team at GetYourRefund.org
       email_subject: Getting your taxes started with GetYourRefund
-      sms_body: "Hello %{preferred_name}, thanks for starting your taxes with GetYourRefund, you’re almost there! Get your tax documents ready and try to complete in one sitting. Remember, we’ll review everything with you before filing so do the best you can. Respond to this text message if you have a question or come back later and upload documents at this secure submission link: %{requested_docs_link} We won’t be able to prepare your taxes until we have all of your required information and tax documents. Then we’ll help you get the tax credits that belong to you!"
+      sms_body: |
+        Hello %{preferred_name}, thanks for starting your taxes with GetYourRefund, you’re almost there! Your case number is %{client_id}. We won’t be able to prepare your taxes until we have all of your required information and tax documents. Upload documents securely here: %{requested_docs_link}
```

Comment:
This says case number but we typically them confirmation numbers. See earlier note. If you're sure about this, fine with me!